app/controllers/people_controller.rb

```diff
@@ -6,19 +6,19 @@ class PeopleController < ApplicationController
   def signin
     #This is the sign-in sheet, not anything about authentication
     @people = Person.active
-    @department = "Police"
+    @department = Department.where(name: "Police")&.first
     @page_title = "Sign-in"
     render :layout => "print_signin"
   end
 
   def orgchart
-    @people = Person.police.active
+    @people = Person.active.select{|person| person.department&.name == "Police"}
     @page_title = "Org Chart"
     render :layout => "orgchart"
   end
 
   def police
-    @people = Person.police.active
+    @people = Person.active.select{|person| person.department&.name == "Police"}
```

Comment:
My previous comment (deleted) about the app-wide concept of Police Department is beyond the scope of this PR, sorry @sergeimellow